ui/hooks/bridge/useBridging.ts

```diff
@@ -52,13 +55,20 @@ const useBridging = () => {
     dispatch(setBridgeFeatureFlags());
   }, [dispatch, setBridgeFeatureFlags]);
 
+  useEffect(() => {
+    isBridgeChain &&
+      isBridgeSupported &&
+      providerConfig &&
+      dispatch(setFromChain(providerConfig.chainId));
+  }, []);
+
```

Comment:
Should anything happen in the negative condition? Like if it's not allowed for bridging, what should happen?

Reply:
Nothing needs to happen if bridging is not allowed. If bridging is supported in the active network but the native extension experience is turned off, `openBridgeExperience` opens Portfolio instead